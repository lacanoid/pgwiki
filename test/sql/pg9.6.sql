\pset null _null_
\pset format unaligned

SET client_min_messages = warning;
SET ROLE postgres;

select ddlx_create(oid) from pg_am where amname = 'btree';
select ddlx_drop(oid) from pg_am where amname = 'btree';

