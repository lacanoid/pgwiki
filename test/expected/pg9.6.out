\pset null _null_
\pset format unaligned
SET client_min_messages = warning;
SET ROLE postgres;
select ddlx_create(oid) from pg_am where amname = 'btree';
ddlx_create
CREATE ACCESS METHOD btree
  TYPE INDEX HANDLER bthandler;

COMMENT ON ACCESS METHOD btree IS 'b-tree index access method';

(1 row)
select ddlx_drop(oid) from pg_am where amname = 'btree';
ddlx_drop
DROP ACCESS METHOD btree;

(1 row)
